addon/src/waiter-manager.ts

```diff
@@ -1,4 +1,4 @@
-import { PendingWaiterState, Waiter, WaiterName } from './types';
```

Comment:
using type imports is important for not shipping too much stuff to the runtime (especially stuff that doesn't exist)